Octokit/Helpers/HttpClientExtensions.cs

```diff
@@ -7,16 +7,12 @@ namespace Octokit
 {
     public static class HttpClientExtensions
     {
-        /// <summary>
-        /// OBSOLETE
-        /// </summary>
-        [Obsolete("This will be removed in a future release")]
-        public static Task<IResponse<T>> Send<T>(this IHttpClient httpClient, IRequest request)
+        public static Task<IResponse> Send(this IHttpClient httpClient, IRequest request)
```

Comment:
I don't think _we_ need it, but it may be useful to people who call `IHttpClient` directly. For example, we use this extension method in GHfW.